meteor/sofie-server/business-logic/services/core-meteor-integration/sofie-core-meteor-rundown-service.ts

```diff
@@ -0,0 +1,87 @@
+import { RundownService } from '../interfaces/rundown-service'
+import { ServerClientAPI } from '../../../../server/api/client'
+import { Meteor } from 'meteor/meteor'
+import { MethodContextAPI } from '../../../../lib/api/methods'
+import { StudioJobs } from '@sofie-automation/corelib/dist/worker/studio'
+import { RundownPlaylistId } from '@sofie-automation/shared-lib/dist/core/model/Ids'
+
+export class SofieCoreMeteorRundownService implements RundownService {
+	public async activateRundown(_rundownId: string): Promise<void> {
+		throw new Error('Not implemented exception')
+	}
+
+	public async takeNext(rundownId: string): Promise<void> {
+		const rundownPlaylistId: RundownPlaylistId = rundownId as unknown as RundownPlaylistId
```

Comment:
Has the input to this method already been saniticed, i.e checked that it has the correct structure?

If so should the type of the input not already be `RundownPlaylistId`?

My guess is that it hasn't been sanitized, due to this being a POC. 
If so, that should propably be done now that we going to make use of it.

Reply:
Refer to https://github.com/tv2/tv-automation-server-core/pull/247#discussion_r1281584423